tests/unit/test_user.py

```diff
@@ -65,3 +66,32 @@ def test_multiple_users_can_be_created():
             return create_unauthorized_user()
 
         assert test_multiple_users_can_be_created()
+
+
+@pytest.mark.usefixtures("empty_test_db")
+class TestOverwrittenAndInheritedMethods:
+    def test_user_save_to_db(self, user_attributes):
+        attrs = UserModel.validate(
+            UserSchema, user_attributes(role=RoleEnum.STAFF.value)
+        )
+        user = UserModel(**attrs)
+        user.save_to_db()
+        lookedup = UserModel.find_by_id(user.id)
+        assert lookedup.password is None
+        user.password = None
+        assert lookedup == user
```

Comment:
What are we testing here?

Reply:
Just making sure we don’t save the password as plain text and getting the user attributes back. I could just test to make sure we don’t save the password.

Reply:
Ah yeah, it might be over testing here.